lib/ansible/modules/network/aci/aci_tenant.py

```diff
@@ -20,161 +20,217 @@
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
-
 module: aci_tenant
-short_description: Direct access to the APIC API
+short_description: Manage tenants on Cisco ACI fabrics
 description:
-    - Offers direct access to the APIC API
-author: Cisco
-extends_documentation_fragment: aci
+- Manage tenants on a Cisco ACI fabric.
+author:
+- Swetha Chunduri (@schunduri)
+- Dag Wieers (@dagwieers)
 requirements:
-    - ACI Fabric 1.0(3f)+
-notes:
+- ACI Fabric 1.0(3f)+
 options:
-    tenant_name:
-        description:
-            - Tenant Name
-        required: true
-        default: null
-        choices: []
-        aliases: []
-    descr:
-        description:
-            - Description for the AEP
-        required: false
-        default: null
-        choices: []
-        aliases: []
+  method:
```

Comment:
We need to get rid of action/method as it does not fit into the "Desired State Configuration" ideology.